commands/new_function_test.go

```diff
@@ -208,6 +210,65 @@ func Test_languageNotExists(t *testing.T) {
 	}
 }
 
+func Test_duplicateFunctionName(t *testing.T) {
```

Comment:
I agree, it's not a best practice, but unsure of how to go about that in a "pure" unit test style.

The trick is that the `runNewFunction` uses the `ioutil.ReadFile()` function to get the data from the file it is attempting to append to. I'm not sure if there's a stub, or some other utility, that would allow us to fake that process without actually writing to the file system.